### ISSUE TITLE
GoTo helper

### ISSUE THREAD

**Issue by DeeDeeRanged** [loki79uk/FS25_FreeLabour#1]
Is it possible just to have the goto helper (driving from A to B) as free labour. I don't mind for them to get paid with field work.
It's just a question of curiosity.

**Comment by loki79uk** [comment]
Yes, if it is just for single player then edit the function here to comment out the free field work, and maybe the root one too:

```lua
function FreeLabour:loadMap(name)
	--print("Loaded Mod: 'FREE LABOUR'")
	
	AIJob.getPricePerMs = Utils.overwrittenFunction(AIJob.getPricePerMs, FreeLabour.getPricePerMs)
	AIJobConveyor.getPricePerMs = Utils.overwrittenFunction(AIJobConveyor.getPricePerMs, FreeLabour.getPricePerMs)
	--AIJobFieldWork.getPricePerMs = Utils.overwrittenFunction(AIJobFieldWork.getPricePerMs, FreeLabour.getPricePerMs)
	
	self.initialised = false
end
```

**Comment by DeeDeeRanged** [comment]
Thx.
Trying it out now, will let you know how it went.

In any case a late Merry Christmas and a Happy Prosperous New Year to you and all your family.

**Comment by DeeDeeRanged** [comment]
A very late reply on how it went. It works great very happy as I use it quite a lot, makes life so much easier lol

I am torn by leaaving this open or closing this issue as it might help others.

In any case I am very grateful of this mod.
Thank you very much.
